test/unit/components/date.js

```diff
@@ -67,26 +70,7 @@ describe('<FormattedDate>', () => {
         expect(rendered.type()).toBe('span');
         expect(rendered.text()).toBe(intl.formatDate(date));
     });
-
```

Comment:
should we not test this anymore?

Reply:
yeah SFC is always re-rendered but since we cache the `Intl.DateTimeFormat` itself this yields minimal benefits (might be even faster cause u don't have to rehash during `shouldComponentUpdate`)

Reply:
The test and the SCU are both required, as this is not being cached.